calicoctl.py

```diff
@@ -237,9 +248,13 @@ def node(ip, node_image):
     if not ipv4_pools:
         print "No IPv4 range defined.  Exiting."
         return
+    ipv6_pools = client.get_ip_pools("v6")
+    if not ipv6_pools:
```

Comment:
Just spotted that we default the pools in master() below. Probably worth a comment here to explicitly call out that we're defaulting the pools elsewhere.